server/routerlicious/packages/lambdas/src/nexus/index.ts

```diff
@@ -642,5 +642,19 @@ export function configureWebSocketServices(
 			}
 			disposers.splice(0, disposers.length);
 		});
+
+		socket.on(
+			"disconnect_document",
+			(clientId: string, documentId: string, errorType?: string) => {
+				if (errorType === undefined) {
+					return;
+				}
```

Comment:
Maybe discussions around this have already happened and I'm just not aware, but I still feel like emitting the event for clean (non-error) disconnect scenarios is unnecessary (and even wasteful, in terms of sending data through the network). Does the server gain anything from seeing `disconnect_document` in non-error cases? The payload might be small, but if it starts coming from every client when browser tabs are closed, feels like it could become a non-insignificant amount of network traffic. I think I still favor `abnormal_disconnect` or `unexpected_disconnect` that always have an error (and the client can just check if the error is not undefined before emitting them). cc @znewton for thoughts.